src/python_testing/matter_testing_infrastructure/chip/testing/commissioning.py

```diff
@@ -21,12 +21,15 @@
 
 import logging
 from dataclasses import dataclass
-from typing import List, Optional
+from typing import Any, List, Optional
 
+import chip.testing.global_stash as global_stash
 from chip import ChipDeviceCtrl, discovery
 from chip.ChipDeviceCtrl import CommissioningParameters
 from chip.clusters import ClusterObjects as ClusterObjects
```

Comment:
Would this work? Or do you need `as ClusterObjects`?

```suggestion
from chip.clusters import ClusterObjects
```

Reply:
Removed `as ClusterObjects` and rebuilt. No issues found on commissioning.